libxtst will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxtst it will fail in the following way:

```
2024/11/12 23:13:02 INFO running step "test/pkgconf"
2024/11/12 23:13:02 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:13:02 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:13:02 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:13:02 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:13:02 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN + basename /home/build/melange-out/libxtst-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN + dev_pkg=libxtst-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN + apk info -L libxtst-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN + grep -q ^Name: usr/lib/pkgconfig/xtst.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN + basename usr/lib/pkgconfig/xtst.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN + lib_name=xtst uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN + echo usr/lib/pkgconfig/xtst.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN + grep -q '^usr/lib/pkgconfig/xtst.pc$\|^usr/share/pkgconfig/xtst.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN + pkgconf --exists xtst uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN + pkgconf --print-errors --exists xtst uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN Package recordproto was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN Perhaps you should add the directory containing `recordproto.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 WARN Package 'recordproto', required by 'xtst', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:02 INFO pod 0d9c6f8fdc17a0f2bde63ec9aac781888d8eebe7386825a0a32cc7623ffa5404 terminated
2024/11/12 23:13:02 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:13:02 INFO   workspace dir: /home/user/tmp/melange-workspace-2703891318
2024/11/12 23:13:02 INFO   guest dir: /home/user/tmp/melange-guest-736775765
2024/11/12 23:13:02 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxtst] Error 1
```

--- a/libxtst.yaml
+++ b/libxtst.yaml
@@ -1,7 +1,7 @@
 package:
   name: libxtst
   version: 1.2.5
-  epoch: 0
+  epoch: 1
   description: X11 Testing -- Resource extension library
   copyright:
     - license: XFree86-1.1
@@ -49,7 +49,11 @@ subpackages:
       runtime:
         - libxtst
         - libxi-dev
+        - xorgproto
     description: libxtst dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libxtst-doc
     pipeline:

